namespaces/live.cloud-platform.service.justice.gov.uk/hale-platform-prod/resources/victimscommissioner-route53.tf

```
@@ -21,3 +21,208 @@ resource "kubernetes_secret" "victimscommissioner_route53_zone_sec" {
     zone_id = aws_route53_zone.victimscommissioner_route53_zone.zone_id
   }
 }
+
+resource "aws_route53_record" "victimscommissioner_route53_a_record" {
+  zone_id = aws_route53_zone.victimscommissioner_route53_zone.zone_id
+  name    = "victimscommissioner.org.uk"
+  type    = "A"
+
+  alias {
+    name                   = "dualstack.jotwp-loadb-1mbwraz503eq6-1769122100.eu-west-2.elb.amazonaws.com."
```

The spreadsheet doesnt have `dualstack` in the front

```suggestion
    name                   = "jotwp-loadb-1mbwraz503eq6-1769122100.eu-west-2.elb.amazonaws.com."
```